client/src/app/site/users/components/user-list/user-list.component.ts

```diff
@@ -69,4 +93,17 @@ export class UserListComponent extends ListViewBaseComponent<ViewUser> implement
     public onPlusButton(): void {
         this.router.navigate(['./new'], { relativeTo: this.route });
     }
+
+    // TODO save all data from the dataSource
+    public saveUserList(): void {
+        this.csvExport.export(
+            this.dataSource.data,
+            [
+                { property: 'full_name', label: 'Name' },
+                { property: 'groups', label: 'Groups' },
+                { property: 'is_active', label: 'Presence' }
```

Comment:
Please use the same csv columns as in 2.3.x:
https://github.com/OpenSlides/OpenSlides/blob/stable/2.3.x/openslides/users/static/js/users/csv.js#L14

Reply:
Excepting 'number' (which is not part of the data I can access in the user list), I included all data now

Reply:
it's named `participant_number`.